win32ss/reactx/ntddraw/dvp.c

```diff
@@ -365,11 +351,10 @@ NtGdiDvpGetVideoPortField(HANDLE hVideoPort,
 
     if (pfnDvpGetVideoPortField == NULL)
     {
-		DPRINT1("Warning: no pfnDvpGetVideoPortField");
+		DPRINT1("Warning: no pfnDvpGetVideoPortField\n");
```

Comment:
tab indent. I will stop mentioning more of them now, even in case I will spot more.